packages/server/src/utilities/templates.js

```diff
@@ -56,6 +56,19 @@ exports.downloadTemplate = async function(type, name) {
   return dirName
 }
 
+async function performDump({ dir, appId, name = "dump.txt" }) {
+  const writeStream = fs.createWriteStream(join(dir, name))
+  // perform couch dump
+  const instanceDb = new CouchDB(appId)
+  await instanceDb.dump(writeStream, {
+    filter: doc => {
+      return !doc._id.startsWith(DocumentTypes.USER)
```

Comment:
Do we need to think about user table relationships? By stripping out the users the linked docs might behave weirdly in the new app, on the other side of the relationship. Might be fine too, but maybe worth checking.